app/controllers/email_notifications/email_notifications_controller.js

```diff
@@ -0,0 +1,170 @@
+'use strict'
+
+// NPM dependencies
+const _ = require('lodash')
+const logger = require('winston')
+
+// Local dependencies
+const response = require('../../utils/response.js').response
+const Email = require('../../models/email.js')
+const paths = require('../../paths.js')
+const CORRELATION_HEADER = require('../../utils/correlation_header.js').CORRELATION_HEADER
+const formattedPathFor = require('../../../app/utils/replace_params_in_path')
+
+// Constants
+const emailConfirmationToggleSubmitLink = formattedPathFor(paths.emailNotifications.confirmation)
```

Comment:
All paths are [by default shared to the template](https://github.com/alphagov/pay-selfservice/blob/master/server.js#L61) so you don’t need to pass them in the controller manually.

You can just use {{ routes.emailNotifications.confirmation }} in the Nunjucks

(I only realised this recently and have been using is since — https://github.com/alphagov/pay-selfservice/blob/master/app/views/payment-links/review.njk#L11-L12

Reply:
Also meant to say you don’t need to use `formattedPathFor` unless the path actually needs formatting aka when it looks something like this `show: '/service/:externalServiceId/team-member/:externalUserId'`